packages/calcite-components/src/components/color-picker/color-picker.e2e.ts

```diff
@@ -2236,9 +2243,64 @@ describe("calcite-color-picker", () => {
       const hueSliderScope = await page.find(`calcite-color-picker >>> .${CSS.hueScope}`);
 
       expect(await hueSliderScope.getComputedStyle()).toMatchObject({
-        top: "10px",
-        left: "0px",
+        top: "9.5px",
+        left: "-0.5px",
       });
     });
   });
+
+  describe("mouse interaction with scope", () => {
+    it("should update value when color field scope is moved", async () => {
+      const page = await newE2EPage();
+      await page.setContent(`<calcite-color-picker ></calcite-color-picker>`);
+      const colorPicker = await page.find("calcite-color-picker");
+
+      const [colorFieldScopeX, colorFieldScopeY] = await getElementXY(
+        page,
+        "calcite-color-picker",
+        `.${CSS.colorFieldScope}`
+      );
+      const value = await colorPicker.getProperty("value");
+
+      await page.mouse.move(colorFieldScopeX, colorFieldScopeY + 0.8);
+      await page.mouse.down();
+      await page.mouse.up();
+      await page.waitForChanges();
+      expect(await colorPicker.getProperty("value")).not.toBe(value);
+    });
+
+    it("should update value when hue scope is moved", async () => {
+      const page = await newE2EPage();
+      await page.setContent(`<calcite-color-picker ></calcite-color-picker>`);
+      const colorPicker = await page.find("calcite-color-picker");
+
+      const [hueScopeX, hueScopeY] = await getElementXY(page, "calcite-color-picker", `.${CSS.hueScope}`);
+      const value = await colorPicker.getProperty("value");
+
+      await page.mouse.move(hueScopeX + 0.8, hueScopeY);
```

Comment:
What does the 0.8 represent? If it's an arbitrary move in pixels, can you simplify this by using whole numbers? Also, can you assign it to a self-documenting const?

This applies to the 10 and 2 values below too.

Reply:
the `0.8` represent the pixel inside the scope element and the reason for using value above 0.5  is to avoid getting rounded off and assert a change in value.